metaci/release/utils.py

```diff
@@ -96,8 +109,10 @@ def send_release_webhook(project_config, release, config_item=None):
 
     payload = {
         "case_template_id": release.change_case_template.case_template_id,
-        "package_name": project_config.project__package__name,
-        "version": project_config.get_version_for_tag(tag),
+        "package_name": release.repo.name,  # Need to figure out.
```

Comment:
I would stick with project_config.project__package__name here; package name is not necessarily the same as repo name.